cmd/mattermost/commands/sampledata.go

```diff
@@ -338,7 +330,7 @@ func sampleDataCmdF(command *cobra.Command, args []string) error {
 		totalUsers := 3 + rand.Intn(3)
 		for len(users) < totalUsers {
 			user := allUsers[rand.Intn(len(allUsers))]
-			if !sliceIncludes(users, user) {
+			if !slices.IncludesString(users, user) {
```

Comment:
I found out that this method already exists in https://github.com/mattermost/mattermost-server/blob/master/utils/utils.go#L13 and therefore you can reference that `utils.StringInSlice` and remove the newly created utils / slices 
... or move all slice related functions from `utils/utils.go` into `utils/slices.go` and keep the new file - either way works for me

maybe it doesnt have to be in this ticket but might be worth doing that later